cmd/satellite/main.go

```diff
@@ -141,6 +142,13 @@ func cmdRun(cmd *cobra.Command, args []string) (err error) {
 		zap.S().Error("Failed to initialize telemetry batcher: ", err)
 	}
 
+	if runCfg.Schema != "" {
+		err = db.CreateSchema(runCfg.Schema)
```

Comment:
Move this into `satellitedb` `CreateTables` method.

Reply:
I tested out adding `CreateSchema` to the `CreateTables` method, but [`CreateTables`](https://github.com/storj/storj/blob/master/satellite/satellitedb/migrate.go#L22) doesn't have the database connstr available to it.  So I'm not sure what is best.  What I initially chose to do is [this](https://github.com/storj/storj/pull/1908/commits/eb0fb725b99132b29c47dab6b8c37d0362938f6f#diff-e7e4cbc943e802fe544ebbf762a3a459R145) just keep the CreateSchema in this file.

Reply:
You can save the schema or connstr in https://github.com/storj/storj/blob/master/satellite/satellitedb/database.go#L47.